### PR TITLE
Add binaries to company

### PR DESCRIPTION
**Pull request by renjith341**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Automated tests included? |No
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | No
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Sales persons should have the ability to add contracts and agreements to a company to ensure that all are having the same information. This PR provides an option to manage company related files from company UI.

#### Steps to test this PR:
1. Create a new company or edit an existing one, multiple files can be added from that user interface.
2. Attached files can be viewed or removed from same user interface.

### CODE REVIEW

File: app/bundles/LeadBundle/Controller/CompanyController.php [hunk 1]
```diff
@@ -771,4 +795,72 @@ public function mergeAction($objectId)
...
         );
     }
+
+    /**
+     * Upload company files.
+     *
+     * @param array   $files
+     * @param Company $company
+     */
+    private function uploadCompanyFiles($files, $company)
```
Comment (escopecz) [inline app/bundles/LeadBundle/Controller/CompanyController.php]: Could you please add the type hints to the method params?

**Comment by dbhurley** [comment]
Hey everyone, I've been mulling this one for a bit here. It feels to me like this is a bit more of a CRM functionality and not really something that fits in marketing automation. I'm not going to close the PR just yet, but I think I'm going to remove it from the 2.13 milestone. I'd like to see us have a bit more discussion about it before we merge it.

**Comment by alanhartless** [comment]
Closing this one as it's quite old. Please consider this as a plugin.

**Pull request closed without merging**